### ISSUE TITLE
Add bibliography support

### ISSUE THREAD

**Issue by giflw** [asciidocfx/AsciidocFX#348]
There is a gem to add bibliography: https://github.com/riboseinc/asciidoctor-bibliography

Would be nice to have it available on the default installation of AsciiDocFX.

**Comment by daacosta** [comment]
Yes... This is needed badly.  If Asciidocfx can use this it will become the monster that I need:

1. Bibliography with support for csl standard
2. BibTex support for styles and the ability to read and use bib files
3. Cross-references
